docs/data_management/lts/tutorial/individual_lts_tutorial.md

```diff
@@ -0,0 +1,113 @@
+conda activate s3
+pip install s3cmd
+```
+
+Once these steps are completed, verify the installation by running `pip list | grep "s3cmd"` in the terminal. If `s3cmd` is listed, as shown in the screenshot below, the package has been successfully installed.
+
+![image](../images/pip-s3cmd.png)
+
+### Install s3cmd on Your Local Systems
+
+To install s3cmd on your local machine, please follow the instructions provided in [our s3cmd documentation for local installation](../../../data_management/lts/interfaces.md/#installation-of-s3cmd-and-s5cmd-on-personal-system).
+
+### Configuring s3cmd for LTS Buckets
+
+Properly configuring `s3cmd` is important for working with LTS buckets and objects. The configuration process varies depending on whether you have a single LTS account or multiple accounts to manage. In this section, we will provide a step-by-step guide tailored specifically for the **Cheaha** system and a researcher with an **individual LTS account**.
+
+Open a terminal using one of the interactive apps on Cheaha. Activate your conda environment created in the [Install s3cmd using within Anaconda Environment](./individual_lts_tutorial.md/#install-s3cmd-within-anaconda-environment-on-cheaha) section, and then run the below command:
+
+```bash
+s3cmd --configure
+```
+
+This will prompt you to enter the access key and secret key associated with your individual LTS account. You will be asked for additional information, which will be displayed on the screen, as shown below. You can copy the necessary details from the example provided [here](../interfaces.md/#configuring-s3cmd).
+
+![image-s3cmd](../images/config-s3cmd.png)
+
+Once the configuration is complete, `s3cmd` will generate a `.s3cfg` file in your home directory (`$HOME`), as shown below. To find your home directory in Cheaha and view the `.s3cfg` file, follow the instructions on our [Navigating Open OnDemand](../../../cheaha/open_ondemand/ood_layout.md/#navigating-open-ondemand) page. Be sure to check the **Show Dotfiles** option in the top right corner to make hidden files visible.
+
+![config-file](../images/s3cfg.png)
+
+### Creating Buckets
+
+Long Term Storage (LTS) services like Amazon S3 uses a specific data organization model based on **buckets** and **objects**. Think of buckets as folders that contain individual pieces of data called objects. Each bucket has a unique name. We have documentation about basic terminology on s3 storage system [here](../index.md/#terminology) and a naming guide for LTS buckets [here](../index.md/#avoiding-duplicate-names-for-buckets).
+
+Once you have complete `s3cmd` configuration, you can create new buckets in your individual LTS storage. To create a bucket use a `mb`(make bucket) command:
+
+```bash
+s3cmd mb s3://your_bucket_name
+```
+
+Please replace `your_bucket_name` with your desired name. This command creates a new bucket that you named `your_bucket_name` in your LTS storage using the currently configured `s3cmd` profile.
```

Comment:
Let's show the researchers what happens when they create a bucket with a non-unique name.

If they try to create a bucket that already exists in their own namespace, it will report success and make no changes.

If they try to create a bucket that already exists in someone else's namespace, it will report error 409.